experiments/AMIP/modular/components/ocean/eisenman_seaice_init.jl

```diff
@@ -226,8 +226,8 @@ function eisenman_seaice_init(
         thermo_params = thermo_params,
         dss_buffer = ClimaCore.Spaces.create_dss_buffer(ClimaCore.Fields.zeros(space)),
     )
-    problem = ODEProblem(ode_function, Y, FT.(tspan), cache)
-    integrator = init(problem, ode_algo, dt = FT(dt), saveat = FT(saveat), adaptive = false)
+    problem = ODEProblem(ode_function, Y, Float64.(tspan), cache)
+    integrator = init(problem, ode_algo, dt = Float64(dt), saveat = Float64(saveat), adaptive = false)
```

Comment:
Why is this change needed to remove OrdinaryDiffEq and DiffEqBase?

Reply:
(Also below)

Reply:
it isn't, but I just noticed we were converting the times to `FT` and figured I would remove that to try to reduce confusion about the type of the time

Reply:
the time float type just gets converted to Float64 internally in ClimaTimeSteppers, so it doesn't matter too much. I just thought it was misleading to explicitly cast it to FT when we don't expect (or want) it to have that type

Reply:
I'm going to make the time FT changes in the PR testing Float32 compatibility more comprehensively, since I think it relates to that more. I'll keep it as a separate commit when I add it there

Reply:
Sounds good! Thank you